### ISSUE TITLE
I can't change the keyboard I'm using on this game

### ISSUE THREAD

**Issue by leviboom09** [CleverRaven/Cataclysm-DDA#40120]
# Describe the bug
Stuck with the security/Default keyboard of my phone

I can't choose other keyboards

# Steps To Reproduce

Just installing from 10533 to 10613 and pressing back button to summon the keyboard

# Expected behavior
You could usually switch to different keyboards on the 10533
there should be an option to change keyboards 
![Screenshot_2020-05-04-11-15-03](https://user-images.githubusercontent.com/59605605/80934427-b66a9480-8dfa-11ea-9b2b-06a1843e4d3e.png)

# Screenshots

# Versions and configuration

# Additional context

**Comment by RedShakespeare** [comment]
Just enter the keyboard settings in **"Settings" of ur cell phone** and deactivate the secure input...
( Thought this might be a common question asked in game forum...? Is it appropriate here... )

**Comment by ZhilkinSerg** [comment]
Do you have any other keyboard apps installed? Are they enabled in "Languages & Input" (or any similar) OS menu?

**Comment by leviboom09** [comment]
> Do you have any other keyboard apps installed? Are they enabled in "Languages & Input" (or any similar) OS menu?

Yes but the option to change the keyboard does not appear on 10613
![IMG_20200504_180948](https://user-images.githubusercontent.com/59605605/80955814-79ba8f80-8e32-11ea-9170-291c3f27321d.png)

**Comment by leviboom09** [comment]
I'll try to update on the latest version if it gets fixed

**Comment by ZhilkinSerg** [comment]
Are you on Samsung device?

**Comment by leviboom09** [comment]
> Are you on Samsung device?

I'm using a Huawei Device

**Comment by AdityaMH** [comment]
> Yes but the option to change the keyboard does not appear on 10613 ![IMG_20200504_180948](https://user-images.githubusercontent.com/59605605/80955814-79ba8f80-8e32-11ea-9170-291c3f27321d.png)

For me the menu always disappear every 3 sec. And i need scroll down the status bar to show up. But scroll down status bar make my character moving to south. That pretty annoying.
